test/statsCases/asset/webpack.config.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
 			{
 				test: /\.(png|jpg|svg)$/,
 				type: "asset"
+			},
+			{
+				test: /\.html$/,
+				type: "asset",
```

Comment:
```suggestion
				type: "asset/resource",
```

otherwise this becomes a DataUrl